handler/openid/flow_hybrid_test.go

```diff
@@ -249,9 +249,22 @@ func TestHybrid_HandleAuthorizeEndpointRequest(t *testing.T) {
 				return makeOpenIDConnectHybridHandler(fosite.MinParameterEntropy)
 			},
 			check: func() {
-				assert.NotEmpty(t, aresp.GetFragment().Get("id_token"))
-				assert.NotEmpty(t, aresp.GetFragment().Get("code"))
-				assert.NotEmpty(t, aresp.GetFragment().Get("access_token"))
+				assert.NotEmpty(t, aresp.GetParameters().Get("id_token"))
+				assert.NotEmpty(t, aresp.GetParameters().Get("code"))
+				assert.NotEmpty(t, aresp.GetParameters().Get("access_token"))
+				assert.Equal(t, time.Now().Add(time.Hour).UTC().Round(time.Second), areq.GetSession().GetExpiresAt(fosite.AuthorizeCode))
+			},
+		},
+		{
+			description: "Default responseMode check",
```

Comment:
We should add a test for the concrete response modes and their expected results (e.g. error)

Reply:
Agree. Added error test case in `authorize_response_writer_test.go`